### DISCUSSION TITLE
I need help I'm getting an error when installing

### DISCUSSION THREAD

**Discussion by darwinikii** [Rapptz/discord.py#6211] [Q&A]
**Python version : 3.8.0**
**The command I used to download : py -3 -m pip install -U discord.py**
**Error code** : Collecting discord.py
  Using cached discord.py-1.5.1-py3-none-any.whl (701 kB)
Collecting aiohttp<3.7.0,>=3.6.0
  Using cached aiohttp-3.6.3.tar.gz (1.1 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\Admın\AppData\Local\Programs\Python\Python38-32\python.exe' 'C:\Users\Admın\AppData\Roaming\Python\Python38\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\ADMN~1\AppData\Local\Temp\tmpsed_mz4g'
         cwd: C:\Users\Admın\AppData\Local\Temp\pip-install-rk0ovvkx\aiohttp_baa5278adbbc41aaba639fd4b805eb48
    Complete output (24 lines):
    running dist_info
    creating C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info
    writing C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\PKG-INFO
    writing dependency_links to C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\dependency_links.txt
    writing requirements to C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\requires.txt
    writing top-level names to C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\top_level.txt
    writing manifest file 'C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\SOURCES.txt'
    reading manifest file 'C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    Error in sitecustomize; set PYTHONVERBOSE for traceback:
    SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0xfd in position 0: invalid start byte (sitecustomize.py, line 7)
    warning: no files found matching 'aiohttp' anywhere in distribution
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.pyd' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
    warning: no previously-included files matching '*.lib' found anywhere in distribution
    warning: no previously-included files matching '*.dll' found anywhere in distribution
    warning: no previously-included files matching '*.a' found anywhere in distribution
    warning: no previously-included files matching '*.obj' found anywhere in distribution
    warning: no previously-included files found matching 'aiohttp\*.html'
    no previously-included directories found matching 'docs\_build'
    writing manifest file 'C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.egg-info\SOURCES.txt'
    creating 'C:\Users\Admın\AppData\Local\Temp\pip-modern-metadata-4rxru40i\aiohttp.dist-info'
    error: invalid command 'bdist_wheel'
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\Admın\AppData\Local\Programs\Python\Python38-32\python.exe' 'C:\Users\Admın\AppData\Roaming\Python\Python38\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\ADMIN\AppData\Local\Temp\tmpsed_mz4g' Check the logs for full command output.
WARNING: You are using pip version 20.3.1; however, version 20.3.3 is available.
You should consider upgrading via the 'C:\Users\Admın\AppData\Local\Programs\Python\Python38-32\python.exe -m pip install --upgrade pip' command.
**(Note : Tried to update the pip sign but still didn't ) 
(Note : Did not work in virtual environment)**

**Comment by Harmon758** [comment]
Duplicate of #5950 

Make sure you have setuptools installed and that it's up to date, e.g. `pip install --upgrade setuptools`.
If it still errors, try installing the latest version of wheel, e.g. `pip install --upgrade wheel`.

For code block usage, see https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks.
